enclave_wrangler/objects_api.py

```diff
@@ -407,6 +407,9 @@ def fetch_cset_and_member_objects(
         cset_versions: List[Dict] = [fetch_cset_version(_id, retain_properties_nesting=True) for _id in codeset_ids]
     else:
         cset_versions: List[Dict] = fetch_objects_since_datetime('OMOPConceptSet', since, verbose)
+    # - Filter any old drafts from sourceApplicationVersion 1.0. Old data model. No containers, etc. See also:
+    #  https://github.com/jhu-bids/TermHub/actions/runs/6489411749/job/17623626419
+    cset_versions = [x for x in cset_versions if x['properties']['conceptSetNameOMOP']]
```

Comment:
The bugfix. See explanation: https://github.com/jhu-bids/TermHub/pull/584#issue-1939046960